### PR TITLE
Windows GitHub actions docs

### PR DESCRIPTION
**Pull request by jtgorman**
some docs on using GitHub Actions

### CODE REVIEW

**Comment by areynold** [comment]
Sorry, Jon, I saw this come in a while ago, but I thought it was on a different repo.

On the whole I think this is fine, but I have some suggestions that I think will make the docs more readable.

My biggest complaint is the side scrolling and lack of syntax highlighting in the preformatted text, but I don't have a ready solution to that problem. I will see what I can turn up.

There's also no way to get to these docs unless you know they're there. Should these be linked from the main index?

Additional comments inline on the file changes.

File: github-actions/index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link rel="styleheet" href="/css/illinois-system.css">
+
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <header class="blog-header my-3 pb-3 border-bottom">
+    <div class="container">
+      <h1>University of Illinois GitHub Service Community Portal - GitHub Actions</h1>
```
Comment (areynold) [inline github-actions/index.html:21]: I think putting the service portal name first obscures the purpose of this page. My vote would be to reverse the order ("Github Actions - University ...") or to use the service portal name only as a breadcrumb in the title element (`<title>Github Actions - University of Illinois...</title>`")

File: github-actions/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    </div>
+  </header>
+  <main class="container">
+    <div class="jumbotron rounded-0">
```
Comment (areynold) [inline github-actions/index.html:25]: I'm not sure the jumbotron is doing readability any favors. You've already got the blog header to introduce the page. At the very least, I would trim this down to a quick introductory paragraph, and leave the meat of the page in a regular container.

File: github-actions/windows/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link rel="styleheet" href="/css/illinois-system.css">
+
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <header class="blog-header my-3 pb-3 border-bottom">
+    <div class="container">
+      <h1>University of Illinois GitHub Service Community Portal - GitHub Actions for Windows</h1>
```
Comment (areynold) [inline github-actions/windows/index.html:21]: Same as previous index. If we're using the service portal name as a breadcrumb component, I think we can drop it from the h1.

File: github-actions/windows/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    </div>
+  </header>
+  <main class="container">
+    <div class="jumbotron rounded-0">
```
Comment (areynold) [inline github-actions/windows/index.html:25]: again, I think we can shrink the jumbotron

**Comment by jtgorman** [comment]
Good suggestion as far as the link on the main page, I'll add that in, will also make the various changes suggested as well in a bit and do another push.

**Comment by areynold** [comment]
I've come across a handful of syntax highlighting projects, but most of them seem to be archival, limited in scope, or both.

The quick and dirty solution would be to use something like [hilite.me](http://hilite.me/) to handle code in pages on a case-by-case basis. I think I'd prefer an optional css class and syntax highlighting library, which could be used by any of our doc authors:
* [Editable text area with syntax highlighted code](https://css-tricks.com/creating-an-editable-textarea-that-supports-syntax-highlighted-code/)
* [Add syntax highlighting to a blog or website](https://www.taniarascia.com/adding-syntax-highlighting-to-code-snippets/)

Happy to move this to an issue if you think it warrants it. I don't think this is enough of a problem to hold up your PR.

File: github-actions/index.html (hunk 1 above)
Comment (jtgorman) [reply github-actions/index.html]: Ok, I went ahead and pulled out the "University ..." out and added breadcrumbs

File: github-actions/index.html (hunk 2 above)
Comment (jtgorman) [reply github-actions/index.html]: Yanked out the jumbotron entirely. Tried a quick intro and given how short the page was it didn't seem worth the extra focus.

File: github-actions/windows/index.html (hunk 3 above)
Comment (jtgorman) [reply github-actions/windows/index.html]: pulled and put in bread crumbs

**Comment by jtgorman** [comment]
Btw, people wanting to see how this will look and don't want to skim the html can look at https://jtgorman.github.io/uillinois-community.github.io/. Not sure if there's another way to visualize the html easily.

I also wasn't entirely sure how we wanted to add links to the front page, so just added a Topics seciton with an unordered list and added the link there.

**Comment by areynold** [comment]
I think that's a reasonable solution. We can reevaluate if the topics start getting unwieldly.

**Pull request merged**